views/check-your-answers.html

```diff
@@ -9,606 +9,202 @@
-                </dt>
-                <dd class="govuk-summary-list__value">
-                    <p>27 March 1987</p>
-                </dd>
-                <dd class="govuk-summary-list__actions">
-                    <a href="/register/managing-officer">
-                        Change
-                        <span class="govuk-visually-hidden"> date of birth</span>
-                    </a>
-                </dd>
-            </div>
-            <div class="govuk-summary-list__row">
-                <dt class="govuk-summary-list__key">
-                    Nationality
-                </dt>
-                <dd class="govuk-summary-list__value">
-                    <p>Canadian</p>
-                </dd>
-                <dd class="govuk-summary-list__actions">
-                    <a href="/register/managing-officer">
-                        Change
-                        <span class="govuk-visually-hidden"> nationality</span>
-                    </a>
-                </dd>
-            </div>
-            <div class="govuk-summary-list__row">
-                <dt class="govuk-summary-list__key">
-                    Home address
-                </dt>
-                <dd class="govuk-summary-list__value">
-                    72 Guild Street
-                    <br>London
-                    <br>SE23 6FH
-                </dd>
-                <dd class="govuk-summary-list__actions">
-                    <a href="/register/managing-officer">
-                        Change
-                        <span class="govuk-visually-hidden"> home address</span>
-                    </a>
-                </dd>
-            </div>
-
-            <div class="govuk-summary-list__row">
-                <dt class="govuk-summary-list__key">
-                    Occupation
-                </dt>
-                <dd class="govuk-summary-list__value">
-                    <p>Director</p>
-                </dd>
-                <dd class="govuk-summary-list__actions">
-                    <a href="/register/managing-officer">
-                        Change
-                        <span class="govuk-visually-hidden"> occupation</span>
-                    </a>
-                </dd>
-            </div>
-            <div class="govuk-summary-list__row">
-                <dt class="govuk-summary-list__key">
-                    Role and responsibilities
-                </dt>
-                <dd class="govuk-summary-list__value">
-                    <p>Strategic management of the company and the issuing of necessary directives</p>
-                </dd>
-                <dd class="govuk-summary-list__actions">
-                    <a href="/register/managing-officer">
-                        Change
-                        <span class="govuk-visually-hidden"> role and responsibilities</span>
-                    </a>
-                </dd>
-            </div>
-        </dl>
+      {% set formattedPrincipalAddressHtml %}
```

Comment:
All fields should be mandatory, we do not need ifs

Reply:
But they are currently defined as optional in the Address model structure @mouhajer-ch , so this looks right to me. Or we change the model definition...?